DataConnectors/Citrix_ADC_WAF.json

```diff
@@ -0,0 +1,116 @@
+{
+    "id": "Citrix",
+    "title": "Citrix ADC WAF",
```

Comment:
Shouldn't this be Citrix Netscaler ADC WAF? If yes, would be great to provide that context

Reply:
To be consistent with title in azure marketplace, I have modified the title to  Citrix WAF (Web App Firewall). Accordingly I have renamed it in all other places.